CONTRIBUTING.md

```diff
@@ -0,0 +1,14 @@
+## For maintaniers
+
+### Publishing a new release
+
+[Note: you must be a member of the [`tedconf`](https://www.npmjs.com/org/tedconf) org to publish to npm]
+
+* `npm version $TYPE -m "message about this version"` where $TYPE indicates the semver release type, eg. `patch`, `major` or `minor`. see the [npm-version docs](https://docs.npmjs.com/cli/version) and [semver docs](http://semver.org/) if you're not sure which applies.
+* `npm publish`
+* `git push --tags`
+* update the [changelog](https://github.com/tedconf/shed-css/blob/master/CHANGELOG.md)
```

Comment:
do you usually update the changelog _after_ pushing the new version? I think it's tidy to update the changelog and push it within the new version so it's tied to it. Thoughts?